packages/flutter/lib/src/cupertino/slider.dart

```diff
@@ -197,11 +197,13 @@ class CupertinoSlider extends StatefulWidget {
   /// Defaults to the [CupertinoTheme]'s primary color if null.
   final Color activeColor;
 
+  /// {@template flutter.cupertino.slider.thumbColor}
   /// The color to use for the thumb of the slider.
   ///
   /// Thumb color must not be null.
```

Comment:
This doesn't appear to be a requirement. Allowing the parameter to be null, to indicate that we should defer to the default is OK.

Reply:
This was not added by me, however your statement is true. Should I remove this "must not be null" line?